server.js

```diff
@@ -349,6 +349,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Wercker V3 integration
```

Comment:
Nothing changed with the old API. See: https://img.shields.io/wercker/ci/54c9168980c7075225004157.svg

The example page uses an ID of an application that no longer exists.

I've added a new route, since the old route is probably still in use, and the new route is backwards compatible. 

You could remove the legacy route from the example page, and just show the new v3 way.